### PR TITLE
STAC-0: Docs for Otel ingress

### PR DESCRIPTION
**Pull request by aacevedoosorio**

### CODE REVIEW

File: setup/install-stackstate/kubernetes_openshift/ingress.md [hunk 1]
```diff
@@ -43,6 +43,40 @@ stackstate \
 stackstate/stackstate-k8s
 ```
 
+## Configure ingress for Open Telemetry Traces via the StackState Helm chart
```
Comment (viliakov) [inline setup/install-stackstate/kubernetes_openshift/ingress.md]:
```suggestion
## Configure Ingress Rule for Open Telemetry Traces via the StackState Helm chart
```
I would call it Ingress Rule explicitly here to exclude any speculations. and later just use Ingress, with the capital 'I'.

File: setup/install-stackstate/kubernetes_openshift/ingress.md [hunk 2]
```diff
@@ -43,6 +43,40 @@ stackstate \
...
+
+The StackState Helm chart exposes an `opentelemetry-collector` section in its values where a dedicated `ingress` can be created. This is disabled by default. The ingress needed for `opentelemetry-collector` purposed needs to support GRPC protocol. The example below shows how to use the Helm chart to configure an nginx-ingress controller with GRPC and  TLS encryption enabled. Note that setting up the controller itself and the certificates is beyond the scope of this document.
```
Comment (viliakov) [inline setup/install-stackstate/kubernetes_openshift/ingress.md]:
```suggestion
The StackState Helm chart exposes an `opentelemetry-collector` service in its values where a dedicated `ingress` can be created. This is disabled by default. The ingress for `opentelemetry-collector` has to support GRPC protocol. The example below shows how to use the Helm chart to configure an nginx-ingress controller with GRPC and  TLS encryption enabled. Note that setting up the controller itself and the certificates is beyond the scope of this document.
```
It is supposed to be a service, right?

File: setup/install-stackstate/kubernetes_openshift/ingress.md [hunk 3]
```diff
@@ -43,6 +43,40 @@ stackstate \
 stackstate/stackstate-k8s
 ```
 
+## Configure Ingress Rule for Open Telemetry Traces via the StackState Helm chart
+
+The StackState Helm chart exposes an `opentelemetry-collector` service in its values where a dedicated `ingress` can be created. This is disabled by default. The ingress needed for `opentelemetry-collector` purposed needs to support GRPC protocol. The example below shows how to use the Helm chart to configure an nginx-ingress controller with GRPC and  TLS encryption enabled. Note that setting up the controller itself and the certificates is beyond the scope of this document.
```
Comment (viliakov) [inline setup/install-stackstate/kubernetes_openshift/ingress.md:48]:
```suggestion
The StackState Helm chart exposes an `opentelemetry-collector` service in its values where a dedicated `Ingress` can be created. This is disabled by default. The Ingress for `opentelemetry-collector` has to support GRPC protocol. The example below shows how to use the Helm chart to configure an nginx-ingress controller with GRPC and  TLS encryption enabled. Note that setting up the controller itself and the certificates is beyond the scope of this document.
```
I just don't get it gramatically.

File: setup/install-stackstate/kubernetes_openshift/ingress.md [hunk 4]
```diff
@@ -43,6 +43,40 @@ stackstate \
...
+
+To configure the opentelemetry-collector ingress for StackState, create a file `ingress_otel_values.yaml` with contents like below. Replace `MY_DOMAIN` with your own domain \(that's linked with your ingress controller\) and set the correct name for the `tls-secret`. Consult the documentation of your ingress controller for the correct annotations to set. All fields below are optional, for example, if no TLS will be used, omit that section but be aware that StackState also doesn't encrypt the traffic.
```
Comment (viliakov) [inline setup/install-stackstate/kubernetes_openshift/ingress.md]:
```suggestion
To configure the `opentelemetry-collector` Ingress for StackState, create a file `ingress_otel_values.yaml` with contents like below. Replace `MY_DOMAIN` with your own domain \(that's linked with your ingress controller\) and set the correct name for the `tls-secret`. Consult the documentation of your Ingress controller for the correct annotations to set. All fields below are optional, for example, if no TLS will be used, omit that section but be aware that StackState also doesn't encrypt the traffic.
```

**Review by craffit — APPROVED** [review]

**Pull request merged**